### ISSUE TITLE
auto apply CKAN format tags

### ISSUE THREAD

**Issue by takurkowski** [ua-snap/ckanext-snap_harvester#27]
any GeoN Online Resource Protocol of type Web Address (URL) should be automatically set as format: html when harvested by CKAN so it previews appropriately.

**Comment by takurkowski** [comment]
this is complete, eh?

**Comment by BobTorgerson** [comment]
Yes, when you harvest, if the resource has a URL field, it should be set to HTML format by the harvester.
